cmd/backup-manager/app/backup/backup.go

```diff
@@ -14,200 +14,128 @@
 package backup
 
 import (
-	"database/sql"
+	"context"
 	"fmt"
-	"io/ioutil"
+	"io"
 	"os/exec"
-	"path/filepath"
-	"strconv"
-	"strings"
-	"time"
 
-	"github.com/mholt/archiver"
-	"github.com/pingcap/tidb-operator/cmd/backup-manager/app/constants"
-	"github.com/pingcap/tidb-operator/cmd/backup-manager/app/util"
+	"github.com/gogo/protobuf/proto"
 	glog "k8s.io/klog"
-)
-
-// BackupOpts contains the input arguments to the backup command
-type BackupOpts struct {
-	Namespace   string
-	BackupName  string
-	Bucket      string
-	Host        string
-	Port        int32
-	Password    string
-	User        string
-	StorageType string
-}
 
-func (bo *BackupOpts) String() string {
-	return fmt.Sprintf("%s/%s", bo.Namespace, bo.BackupName)
-}
+	kvbackup "github.com/pingcap/kvproto/pkg/backup"
+	"github.com/pingcap/tidb-operator/cmd/backup-manager/app/util"
+	"github.com/pingcap/tidb-operator/pkg/apis/pingcap/v1alpha1"
+)
 
-func (bo *BackupOpts) getBackupFullPath() string {
-	return filepath.Join(constants.BackupRootPath, bo.getBackupRelativePath())
-}
+const (
+	// MetaFile represents file name
+	MetaFile = "backupmeta"
+)
```

Comment:
All constants are defined [here](https://github.com/pingcap/tidb-operator/blob/master/cmd/backup-manager/app/constants/constants.go), can we add more detailed comments to indicate that this variable is for `br`?

Reply:
updated